### DISCUSSION TITLE
Converting from `LocalDateTime` to `LocalDate` uses `XMLGregorianCalendar`

### DISCUSSION THREAD

**Discussion by Sirttas** [mapstruct/mapstruct#3169] [Q&A] [answered]
I don't know if this is a bug or an intentional behavior, but it feels a bit weird and a bit heavy for a simple task.

Edit: I also realized that its not even working. The date end up being null.

**Comment by filiphr** [comment, marked as answer]
@Sirttas have a look at https://github.com/mapstruct/mapstruct/issues/3199. We never had out-of-the-box support for converting `LocalDateTime` to `LocalDate`. It worked by accident because we have support for converting each of those to `XMLGregorianCalendar`
